Xamarin.PropertyEditing.Mac/Controls/ThicknessEditorControl.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using AppKit;
+using CoreGraphics;
+using Xamarin.PropertyEditing.Drawing;
+using Xamarin.PropertyEditing.Mac.Resources;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal abstract class ThicknessEditorControl<T> : BaseRectangleEditorControl<T>
+	{
+		public ThicknessEditorControl ()
+		{
+			// TODO localize
+			XLabel.Frame = new CGRect (28, 27, 50, 22);
+			XLabel.Font = NSFont.FromFontName (DefaultFontName, DefaultDescriptionLabelFontSize);
+			XLabel.StringValue = "LEFT";
```

Comment:
Why would we have a todo for localization when we can do it now?

Reply:
The comment was copy and pasted in and for all the those types of controls, we never Localised, X, Y, Width or Height as the IDEs don't localise those, they are kept in English, IIRC after chatting to @alanmcgovern. Happy to localise all these if I've misunderstood what needed to be localised and what didn't.